web/mygame/scenes/startup.txt

```diff
@@ -18,10 +18,7 @@
 *title My First ChoiceScript Game
 *author Anonymous
 *scene_list
-  startup
-  animal
-  variables
-  gosub
+  test

```

Comment:
Nothing in scenes was intended to be changed - it must have somehow gotten auto-committed when I forked and updated with my local branch.